sdk/js/src/Request.ts

```diff
@@ -6,13 +6,12 @@ import axiosRetry from 'axios-retry'
 import { BucketedUserConfig, DVCLogger } from '@devcycle/types'
 
 const axiosClient = axios.create({
+    timeout: 60 * 1000,
     validateStatus: (status: number) => status < 400 && status >= 200,
```

Comment:
The one problem for this with `status < 400` is that 404s for invalid configs would get retried...

Reply:
isn't that what retryCondition is for? also pretty sure that case is a 401 Unauthorized